BuildForSDG/team-010-backend
# fix(setup ci|cd): setup ci|cd pipelines

#### What does this PR do?
- fixes setup ci|cd pipelines

#### Description of Task to be completed?
- find heroku log
- configures package.json
- configures .travis.yml

#### How should this be manually tested?
- Check heroku staging - https://staged-team-010-backend.herokuapp.com/
- check travisCI build - https://travis-ci.com/github/BuildForSDG/team-010-backend/builds/166455125
- check coveralls coverage - https://coveralls.io/github/BuildForSDG/team-010-backend

#### Any background context you want to provide?
- Please take note that all test env variable would henceforth be added on .travis.yml file
- ignore the currently unkown badges on the branch, the badges can only reflect the develop branch which would begin after merge

#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
#### Questions:
